ui/page/root/home_page.go

```diff
@@ -415,11 +424,19 @@ func (hp *HomePage) LayoutTopBar(gtx C) D {
 		Padding: layout.Inset{
 			Right:  padding20,
 			Left:   padding20,
-			Top:    padding10,
-			Bottom: padding10,
+			Top:    topBottomPadding,
+			Bottom: topBottomPadding,
 		},
 	}.Layout(gtx,
-		layout.Rigid(hp.totalBalanceLayout),
+		layout.Rigid(func(gtx C) D {
+			// Hide the total asset balance usd amount while on mobile view
+			// and on the SingleWalletMasterPage.
+			if hp.CurrentPageID() == MainPageID {
```

Comment:
how do you know current app is mobile view ? I don't see any check mobile view for it

Reply:
the mobile view comment shouldn't be there, thanks